packages/jest-cli/src/init/questions.ts

```diff
@@ -8,6 +8,12 @@
 import type {PromptObject} from 'prompts';
 
 const defaultQuestions: Array<PromptObject> = [
+  {
```

Comment:
adding a test for this in https://github.com/facebook/jest/blob/d63f18a80909db1c1c2ee29e599ed1b03fad583d/packages/jest-cli/src/init/__tests__/init.test.js would be awesome

Reply:
@SimenB I've added a `has-jest-config-file-ts` folder which will be covered by this [test](https://github.com/facebook/jest/blob/d63f18a80909db1c1c2ee29e599ed1b03fad583d/packages/jest-cli/src/init/__tests__/init.test.js#L167). Do you see something extra that we can add?

Reply:
Yeah, a test asserting a file is written to disk with the correct extension